src/Core/RuleHelpLinkProvider.cs

```diff
@@ -37,7 +37,11 @@ public class RuleHelpLinkProvider : IRuleHelpLinkProvider
             { SonarRuleRepoKeys.CSharpRules, "csharp" },
 
             { SonarRuleRepoKeys.JsSecurityRules, "javascript" },
-            { SonarRuleRepoKeys.TsSecurityRules, "typescript" }
+            { SonarRuleRepoKeys.TsSecurityRules, "typescript" },
+
+            // TODO: there may be other repo keys that have different language names on rules.sonarsource.com
+            // See https://github.com/SonarSource/sonarlint-visualstudio/issues/4586.
+            { SonarRuleRepoKeys.HtmlRules, "html" }
```

Comment:
Tells the rule help link builder the "folder" name to use in the rules.sonarsource.com URL.